test/common/testlib.py

```diff
@@ -2092,6 +2092,33 @@ def enable_root_login(self):
         if not self.machine.ostree_image and self.file_exists(disallowed_conf):
             self.sed_file('/root/d', disallowed_conf)
 
+    def setup_provisioned_hosts(self, disable_preload=False):
+        """Setup provisioned hosts for testing
+
+        This sets the hostname of all machines to the name given in the
+        provision dictionary and optionally disabled preload when it runs
+        on a development build.
+        """
+        host_entries = ""
+        for name, m in self.provision.items():
+            print(m)
```

Comment:
Debugging leftover

Reply:
That and `host_entries` is totally unused